src/js/main.js

```diff
@@ -44,12 +58,97 @@ Site = {
 
 };
 
+Site.Index = {
+  init: function() {
+    var _this = this;
+
+    if (window.location.href.indexOf('#shop') !== -1) {
+      // url has #shop
+
+      // remove #shop from url in address bar
+      history.replaceState({}, document.title, '/');
+
+      // scrollto shop after 300ms
+      setTimeout(function(){
+        _this.scrollToShop();
+      }, 300);
+    }
+
+    if ($('.index-header').length) {
+      // bind stickyheader and scrollto shop
+      _this.getHeaderTop();
+      _this.bindSticky();
+      _this.bindScroll();
+    }
+
+    // if youtube video in splash, detect autoplay
+    // if mobile browser detected, remove youtube video
+    if ($('#front-video').length) {
+      detectAutoplay(2000, function() {
+        $('#front-video').remove();
+      });
+    }
+  },
+
+  getHeaderTop: function() {
+    var _this = this;
+
+    _this.headerTop = $(window).innerHeight() - $('#header').outerHeight();
+
+    _this.setSticky();
+  },
+
+  bindSticky: function() {
+    var _this = this;
+
+    $(window).on('scroll', function() {
+      _this.setSticky();
+    });
+  },
+
+  setSticky: function() {
+    var _this = this;
+    var scrollPos = $(window).scrollTop();
+
+    // stick header
+    if (scrollPos >= _this.headerTop){
+      $('#header').removeClass('index-header');
+    }
+    // unstick header
+    if (scrollPos < _this.headerTop){
+      $('#header').addClass('index-header');
+    }
+  },
+
+  bindScroll: function() {
```

Comment:
this is a confusing function name as I think it is binding something to scroll event not a click